src/App.jsx

```diff
@@ -0,0 +1,53 @@
+/* eslint linebreak-style: ["error", "windows"] */
+import {
+  BrowserRouter as Router,
+  Routes,
+  Route,
+} from 'react-router-dom';
+import Navbar from './components/navbar/Navbar';
+import AboutScreen from './routes/About';
+import AdminScreen from './routes/admin/Admin';
+import BookingScreen from './routes/Booking';
+import DetailScreen from './routes/Detail';
+import EditScreen from './routes/admin/Edit';
+import HomeScreen from './routes/Home';
+import NewScreen from './routes/admin/New';
+import ReservationScreen from './routes/Reservation';
+import LoginScreen from './routes/Login';
+import SignupScreen from './routes/Signup';
+import ProtectedRoutes from './routes/ProtectedRoutes';
+
+function App() {

```

Comment:
- [ ] I wasn't able to create a user:

![image](https://user-images.githubusercontent.com/57421823/203972672-e6501d13-30d3-4a31-9e50-7189af5e081a.png)

I suspect this is the problem:

![image](https://user-images.githubusercontent.com/57421823/203972804-e38a7f20-96f3-4288-b4b6-fdc175c17b0e.png)

![image](https://user-images.githubusercontent.com/57421823/203972847-d38445c5-618e-4267-8a88-ef8cbb1a9c45.png)

On one file, you refer to localhost:3001 as the base url.  On the other file (the component) localhost:3000 is being used.  

Is this is intended and your app needs special procedures to be run correctly, kindly add them to your readme.  If this is a bug, kindly fix it so your app runs as intended.